app/views/dashboard/_course_row.html.haml

```diff
@@ -8,8 +8,8 @@
       .course-details_title
         = course_i18n('instructors', course)
       .course-details_value
-        - course.instructors.each do |user|
-          = user.username
+        - course.instructors.first(1).each do |courses_users|
```

Comment:
I don't think this does what you're intending. Course.instructors.first will still fetch a User record, not a CoursesUsers record.

Also, #1459 is referring to the wiki pages that get posted from the Dashboard to Wikipedia — like [this one](https://en.wikipedia.org/wiki/Wikipedia:Wiki_Ed/Carleton_University/Colonialism_and_Post-colonialism_(Fall_2017)) — which gets generated by `lib/wiki_course_output.rb`. The place to intervene for #1459 will be to change how `@first_instructor` gets set in that class.